src/aggregated_bls/test.rs

```diff
@@ -28,3 +28,69 @@ fn agg_sig_test_3() {
     assert_ne!(bls_sig.verify(&message, &p1_keys.pk_i), true);
     assert_ne!(bls_sig.verify(&[10, 11, 12], &apk), true);
 }
+
+// test batch 3 out of 3 for 3 messages
+#[test]
+fn agg_sig_test_3_batch_3() {
+    let key_vec: Vec<Vec<_>> = (0..3)
+        .map(|_| (0..3).map(|j| Keys::new(j)).collect())
+        .collect();
+
+    let pk_vec: Vec<Vec<_>> = (0..3)
+        .map(|i| (0..3).map(|j| key_vec[i][j].pk_i).collect())
+        .collect();
+
+    let apk_vec: Vec<_> = (0..3).map(|i| Keys::aggregate(&pk_vec[i])).collect();
+
+    let msg_vec = vec![[1].as_ref(), [2].as_ref(), [3].as_ref()];
+
+    let sig_vec: Vec<Vec<_>> = (0..3)
+        .map(|i| {
+            (0..3)
+                .map(|j| key_vec[i][j].local_sign(msg_vec[i], &pk_vec[i]))
+                .collect()
+        })
+        .collect();
+
+    let bls_sig_vec: Vec<_> = (0..3)
+        .map(|i| Keys::combine_local_signatures(&sig_vec[i]))
+        .collect();
+
+    let bls_sig_agg = Keys::aggregate_bls(&bls_sig_vec);
+
+    assert_eq!(
+        Keys::aggregate_verify(&apk_vec, &msg_vec, &bls_sig_agg),
+        true
+    );
+    assert_eq!(
```

Comment:
can you add a comment for the test cases that fails? (i.e. this should fail because apk is wrong)